ci-scripts/ci-runner.bash

```diff
@@ -134,7 +134,15 @@ if [ "X${MODULEUSE}" != "X" ]; then
     module use ${MODULEUSE}
 fi
 
-module load reframe
+if [[ $(hostname) =~ tsa ]]; then
+    # FIXME: Temporary workaround until we have a reframe module on Tsa
+    module load python
+    python3 -m venv.unittests
```

Comment:
this needs to change to:
python3  -m venv venv_unittests
source venv_unittests/bin/activate

Reply:
I will fix it but won't change the `.` to `_`.